dali/test/python/operator/test_remap.py

```diff
@@ -122,6 +122,32 @@ def test_remap(self, map_mode):
                            **self.common_dali_pipe_params)
         self._compare_pipelines_pixelwise(dpipe, cpipe, N_iterations=2, eps=.01)
 
+    def benchmark_remap_against_cv(self, map_mode):
+        import torch.cuda.nvtx as nvtx
```

Comment:
Do those test run with nose? If so, we should not use torch unless we mark it as such with the existing mechanism to mark FW dependent tests

Reply:
These test is not run with nose. I used this for benchmarking, so I left it there.